docs/src/docs/asciidoc/user/107.adoc

```diff
@@ -9,25 +9,61 @@ ifdef::notBuildingForSite[]
 include::menu.adoc[]
 endif::notBuildingForSite[]
 
-== Getting started with Ehcache 3.0 & JSR-107
+== JCache overview
 
-There is nothing special for you to do to use Ehcache 3.0 as the caching provider for your application: add the
-`ehcache-3.0.0.jar` to your application's classpath (possibly removing the previous provider's jar) and you are ready to
-go:
+JCache (aka JSR-107) specification defines the standard caching API for Java.
+The specification was developed under the Java Community Process v2.9 by an expert group including members from the Ehcache developer community.
+Hence being one of the pioneers in the Java caching domain, Ehcache 3 also offers an implementation that is fully compliant with the JCache specification.
+JCache provides a very simple API set that is easy to use and vendor neutral.
+
+For years the biggest problems that application developers have faced while wanting to try cache implementations by different vendors is the stark contrast in the APIs offered by these vendors.
+Developers were forced to rewrite a whole lot of their application code just to try out out a new caching solution.
+This lead to a lot of developers remain stuck with what they had even when they knew that there is a better product out there.
+JCache specification was the biggest boon for those developers as now there is a Java standard caching API that they can use.
+So it is fairly easier for an application developer to switch between the products by different vendors and choose the one that suits them best without changing a single line of their application code.
```

Comment:
drop _fairly_, "between products by different vendors", "of their application code interacting with caches."